src/Microsoft.IdentityModel.Tokens/Validation/Results/TokenValidationUnit.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    /// <summary>
+    /// Unit type used to represent an empty value in ValidationResults.
+    /// </summary>
+    /// <remarks>All copies of TokenValidationUnit are considered equal.</remarks>
+    internal record struct TokenValidationUnit
```

Comment:
Not sure if using the term `unit` is very descriptive for this intent?

Reply:
Given the C# language does not implement a Unit type (https://en.wikipedia.org/wiki/Unit_type), we need to resort to creating our own. But at the same time I am trying to be wary and future-proof of simply calling this type `Unit` in case other libraries or even the runtime decide to implement it as well.
If we want to rename it to something else, my only requirement is that we keep the `Unit` in the name to make it clear to anyone familiar with the concept of what it is.

Reply:
Makes sense, I ok with this name for now. We can discuss a name change if others find it too general. Thanks for the explanation.